qa/rpc-tests/bigblocks.py

```diff
@@ -151,12 +155,17 @@ def copy_mempool(self, from_node, to_node):
             txdata = from_node.getrawtransaction(txid)
             to_node.sendrawtransaction(txdata)
 
-    def TestMineBig(self, expect_big):
+    def TestMineBig(self, expect_big, expect_version=None):
         # Test if node0 will mine big blocks.
         b1hash = self.nodes[0].generate(1)[0]
         b1 = self.nodes[0].getblock(b1hash, True)
         assert(self.sync_blocks(self.nodes))
 
+        if expect_version:
+            assert b1['version'] & FORK_BLOCK_BIT
+        elif not expect_version==None:
+            assert not b1['version'] & FORK_BLOCK_BIT
+
         if expect_big:
             assert(b1['size'] > 1000*1000)
```

Comment:
Got that, thanks. I was looking at it from usual behaviour as opposed to specific test case behaviour.